Empire/Empire.h

```diff
@@ -376,6 +379,7 @@ class FO_COMMON_API Empire {
     int                             m_id = ALL_EMPIRES;         ///< Empire's unique numeric id
     std::string                     m_name;                     ///< Empire's name
     std::string                     m_player_name;              ///< Empire's Player's name
+    bool                            m_authenticated;            ///< Empire's Player's authentication status
```

Comment:
maybe this is the wrong thing to call this flag? It's not so much that the empire (or the player) is authenticated, but rather that the empire can only be played by the player who is authenticated with that name.

Reply:
Added more documentation here.